source/Core/Field.php

```diff
@@ -60,14 +62,9 @@ public function __construct($value = null, $type = self::T_TEXT)
     {
         // duplicate content here is needed for performance.
         // as this function is called *many* (a lot) times, it is crucial to be fast here!
-        switch ($type) {
-            case self::T_TEXT:
-            default:
-                $this->rawValue = $value;
-                break;
-            case self::T_RAW:
-                $this->value = $value;
-                break;
+        $this->rawValue = $value;
```

Comment:
Looks like this point should go to "else", no?

Reply:
I think it's not needed, because of the behavior of the `__get` function.

Reply:
In case of get('value'), the htmlspecialchars part of getter will be additionally called if we constructed as RAW. Have i missed something? Not sure if its good or bad, but the flow looks not as it was before anymore :)

Reply:
The htmlspecialchars will be called everytime get('value') is called anyway.
We did change the flow of setting just a little bit to save some time, but it doesn't matter because the _get('value') isn't affected at all.

Reply:
@Sieg to me it looks like internal flow changed a bit, but outside behavior stayed the same

Reply:
Yah, looks like i have missed that those are magic methods and are not called when we already have those values filled. But here is another question - We have _initValue method, which basically is for initializing the value, maybe it would be clever to use that one here?